### DISCUSSION TITLE
Possible to move subtasks

### DISCUSSION THREAD

**Discussion by Rorschach1010** [TechbeeAT/jtxBoard#525] [Q&A] [answered]
I'm not sure if this isn't possible at all or I'm missing something. I'm planning to move from Tasks.org to JTX. What I'm looking for is if it's possible to move subtasks into different tasks. With the tasks.org app, you can just touch and pull a task/subtask into a different task or even make it a subtask of another subtask.
Seems like it's not possible to do this conveniently with a small finger move on JTX. However, I can't find an option for it anywhere in the settings either. So once created, you can't move any tasks at all?

**Comment by patrickunterwegs** [comment, marked as answer]
Hey @Rorschach1010, currently you have the option to  mark entries in the list and through the update entries function you can choose a parent to link to. You can also do the other way round and edit an entry (with the subtasks section visible) and use the button with a chain and "+" to link another subtask.

Regrettably drag and drop is not possible. This is more about the fact that there is no good library that would properly support such a functionality for Jetpack Compose (the UI-framework jtx Board uses)

**Comment by Rorschach1010** [comment]
Wow, I found it but to be fairly honest, I would have never found it myself. It's sad to hear that drag and drop isn't easily doable but I feel like it should at least be doable inside of a tasks setting. I hope you consider my feature request.

In any way, my question is answered as moving them is indeed possible. Thanks a bunch.

**Reply by patrickunterwegs** [reply]
Indeed, this Jetpack Compose has many advantages, but the major disadvantage is really that some things are just not available yet. I'm checking regularly for updates or libraries, but so far everything that I have tried didn't work properly and caused more problems than it solved. But yes, I'll keep an eye on that and will surely implement drag and drop once there is a proper way to do it! 😉
